### DISCUSSION TITLE
Orders field type

### DISCUSSION THREAD

**Discussion by marionnewlevant** [craftcms/commerce#2393] [Ideas]
**What are you trying to do?**
I have an entry which needs to link to an Order. Currently using the order id, but a proper relation would be nice

**What's your proposed solution?**
A 'Commerce Orders' field extending BaseRelationField. I might do it as a plugin.

**Comment by bossanova808** [comment]
@marionnewlevant came up in the commerce 3 channel recently. Is a common request I think. A plugin or even PR would be good. Would be great if there field showed more than just the id of the order (e.g. the billing name)...

**Comment by lukeholder** [comment]
Good idea. Added it to the list.

**Reply by goraxan** [reply]
Was this finally implemented? I can't find it anywhere
